spec.html

```diff
@@ -35770,6 +35770,62 @@ <h1>Array.prototype.findIndex ( _predicate_ [ , _thisArg_ ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.prototype.findlast">
+        <h1>Array.prototype.findLast ( _predicate_ [ , _thisArg_ ] )</h1>
+        <emu-note>
+          <p>_predicate_ should be a function that accepts three arguments and returns a value that is coercible to a Boolean value. `findLast` calls _predicate_ once for each element of the array, in descending order, until it finds one where _predicate_ returns *true*. If such an element is found, `findLast` immediately returns that element value. Otherwise, `findLast` returns *undefined*.</p>
+          <p>If a _thisArg_ parameter is provided, it will be used as the *this* value for each invocation of _predicate_. If it is not provided, *undefined* is used instead.</p>
+          <p>_predicate_ is called with three arguments: the value of the element, the index of the element, and the object being traversed.</p>
```

Comment:
```suggestion
          <p>_predicate_ is called with three arguments: the element, the index of the element in the array, and the object being traversed.</p>
```

Just to be super clear that we are not keeping a counter during traversal.

Reply:
Done.